src/presentation/cli/controller/cron.go

```diff
@@ -0,0 +1,123 @@
+package cliController
+
+import (
+	cliHelper "github.com/speedianet/os/src/presentation/cli/helper"
+	"github.com/speedianet/os/src/presentation/service"
+	"github.com/spf13/cobra"
+)
+
+type CronController struct {
+	cronService *service.CronService
+}
+
+func NewCronController() *CronController {
+	return &CronController{
+		cronService: service.NewCronService(),
+	}
+}
+
+func (controller *CronController) Read() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
```

Comment:
Why cron is list while the others are get?

Reply:
Done.